en/docs/install-and-setup/setup/api-controller/managing-microgateways/managing-microgateways-with-ctl.md

```diff
@@ -0,0 +1,296 @@
+# Managing Microgateways
+
+WSO2 API Controller (apictl) can be used as a developer CLI tool for Microgateway. It enables interacting with one or more Microgateways without having to perform the following tasks via the WSO2 API Manager (WSO2 API-M).
+
+- Deploy an API
+- View the list of deployed APIs
+- Undeploy an API
+
+!!! info
+    **Before you begin** 
+
+    -  Download the Microgateway distribution from the [github release page's](https://github.com/wso2/product-microgateway/releases) and follow the Quick Start Guide attached with the release.
+
+    -  Make sure the apictl is downloaded and initialized, if not follow the steps in [Download and Initialize the CTL Tool]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-ctl-tool). Following the guide upto [Check the version of the CTL]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#check-the-version-of-the-ctl) will be sufficient to continue. 
+
+
+## Add a Microgateway Environment
+
+A microgateway here refers to the entire cluster which includes all three components: adapter, router and enforcer. By adding a microgateway environment, the url to the adapter will be stored in a config file, and therefore enables login. 
+
+Once you have added a microgateway environment, and logged in, other commands such as `apictl mg deploy api` can be used by only specifying an environment. This allows you use such commands without entering the microgateway adapter url, username and password everytime.
```

Comment:
Shall we change this as like this?
```suggestion
Once you have added a microgateway environment, and logged in, other commands such as `apictl mg deploy api` can be used by only specifying an environment. This allows you to use such commands without entering the microgateway adapter url, username, and password every time.
```

Reply:
shall we use "all the time" instead of "every time"

Reply:
Adding the suggested changes. When the meaning of every time is googled, it says 'without exception'. Therefore I think it is more suitable.